docs/developer-docs/gas-cost.md

```diff
@@ -83,18 +85,18 @@ Note that query messages are currently free, but this may change in the future.
 
```

Comment:
Comment was about the sentence "An additional factor of cost includes the number of nodes in the subnet the canister is deployed on.":

Is this scaling factor not applicable to all operations? If so, we might want to state that in the beginning of the section. For example after the sentence "Each type of canister operation and resource has a different cycles cost associated with it, and the canister responsible for paying the cycles varies based on the type of operation."
we could write:
"The following subsections explain the pricing for the different operation and resource types for a 13 node subnet. The same operations and resources cost linearly more on subnets with more nodes."

Reply:
Removed "An additional factor of cost includes the number of nodes in the subnet the canister is deployed on."

Added "The following subsections explain the pricing for the different operation and resource types for a 13 node subnet. The same operations and resources cost linearly more on subnets with more nodes."